test/e2e/bootstrapper_test.go

```diff
@@ -35,15 +39,30 @@ func TestBootstrapper(t *testing.T) {
 	// TODO: Consider doing the same with kubelet. We can either provide our own or download it from the internet
 	// 		 if we choose to download it, we will have to compare expected vs actual SHA hashes for security reasons
 	ensureIgnitionFileExists(t, ignitionFilePath)
+
+	// Stop the kubelet service so we can remove the kubelet logfile
 	wmcb, err := bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
 	assert.Nilf(t, err, "Could not create WinNodeBootstrapper: %s", err)
+	wmcb.StopAndRemoveServices()
```

Comment:
So technically if you run this test, it will disrupt the running kubelet service?